### ISSUE TITLE
use tf.variance_scaling_initializer will lead to NaN

### ISSUE THREAD

**Issue by tenggyut** [YunYang1994/tensorflow-yolov3#441]
Hi, I am trying to rewrite your repo with tensorpack. One interesting issue came up during the process:

When using tf.variance_scaling_initializer to init the detection head(the backbone is efficientnet-b0 with pretraining), all loss become NaN after a few training steps. Once I switch to your default initializer(random_normal_initializer) this issue goes away....

variance_scaling_initializer surpose to be more effective I think, as many papers、dl frameworks use this as defaults. Any Idea?
